### ISSUE TITLE
Investigate Aurum

### ISSUE THREAD

**Issue by alanhdu** [adi-foundry/nycodex#4]
Look at https://github.com/mitdbg/aurum-datadiscovery and see what features it supports and how to use it.

**Comment by alanhdu** [comment]
So our conclusion: Aurum is a super cool technology, but it's I don't think it's production-ready. There build-step is fairly complicated and it relies on a running Elasticsearch instance. As part of the core inference, we're not comfortable running an Elasticsearch instance along with our main PostgreSQL database, and we're also not comfortable trying to adapt to any API changes that might happen on the Aurum side.

In short: Aurum is definitely something we should look at for offline batch analysis, but probably isn't suitable for online data ingestion.
